run.sh

```diff
@@ -112,6 +112,9 @@ generate_user_properties_file() {
 # If the user didn't supply their own user.properties file, generate it
 $SUMO_GENERATE_USER_PROPERTIES && {
```

Comment:
what if the user supplies their own user.properties file (`SUMO_GENERATE_USER_PROPERTIES` is false), I don't think this added code block to check for the FIPS flag will be run in that case?